src/Framework/PropertyReassignmentEventArgs.cs

```diff
@@ -73,12 +105,8 @@ public override string Message
         {
             get
             {
-                if (RawMessage == null)
-                {
-                    RawMessage = FormatResourceStringIgnoreCodeAndKeyword("PropertyReassignment", PropertyName, NewValue, PreviousValue, Location);
-                }
-
-                return RawMessage;
+                string formattedLocation = File != null ? $"{File} ({LineNumber},{ColumnNumber})" : Location;
+                return string.Format(RawMessage, PropertyName, NewValue, PreviousValue, formattedLocation);
```

Comment:
Same, look into caching the formatted result and look into getting the resource string here.